tamago-notification/src/main/kotlin/tamago/server/notification/FirebaseConfig.kt

```
@@ -42,6 +44,13 @@ class FirebaseConfig(
         return FirebaseApp.initializeApp(options)
     }
 
+    private fun openCredentialsStream(path: String): InputStream =
+        try {
+            Files.newInputStream(Paths.get(path))
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)